pom.xml

```diff
@@ -204,13 +204,13 @@
             <version>7.0</version>
             <scope>provided</scope>
         </dependency>
-
-        <dependency>
-            <groupId>org.glassfish.jersey.media</groupId>
-            <artifactId>jersey-media-json-jackson</artifactId>
-            <version>2.0</version>
-            <scope>provided</scope>
-        </dependency>
+		
```

Comment:
Can you verify the indentation (seems tabs are used in place of spaces). There is the .editorconfig in the root directory of the project to standardize the config (it should be updated to set the indentsize of 2 for *.xml).